runelite-client/src/main/java/net/runelite/client/ui/ClientUI.java

```diff
@@ -688,41 +698,92 @@ private void contract()
 		pluginPanel = null;
 	}
 
-	private void expandFrameBy(final int value)
+	private void expandFrameBy(int value)
```

Comment:
You probably should not change the provided value, instead just make temporary one inside the method and change that one